colcon-build/action.yaml

```
@@ -38,6 +38,10 @@ inputs:
     description: Will be exported as MAKEFLAGS environment variable. e.g. "-j 4"
     default: ""
     required: false
+  overlay-workspace:
```

Thanks for the update, but I feel more natural to name it "underlay-workspace" since this is the path to underlay workspace against the colcon build target. 

```suggestion
  underlay-workspace:
```

---

https://colcon.readthedocs.io/en/released/user/using-multiple-workspaces.html#using-multiple-independent-workspaces

> If you have multiple independent workspaces, then they can be used at the same time by sourcing them in sequence. Workspaces are independent if neither workspace has a package that depends on a package in the other workspace.
> ```
> source foo_ws/install/setup.bash
> source bar_ws/install/setup.bash
> ```
> The first workspace foo_ws is called the underlay workspace. The second, bar_ws, is called the overlay workspace. When sourcing more then two workspaces each workspace is said to overlay the previous one.

---

Sounds good. https://github.com/autowarefoundation/autoware-github-actions/pull/337/commits/008569a234534ebc7e67119a2b5ae9673fbad76c